src/cage/src/memory/vmmap.rs

```diff
@@ -270,6 +270,16 @@ impl Vmmap {
             program_break: 0,
         }
     }
+    
+    // Clear the vmmap struct, used for exec syscall
```

Comment:
probably need a bit more of "why" were doing it this way since this is a design decision